### PR TITLE
EOS:16667 [DO NOT MERGE] Build and README Integration of all s3cortxutils (s3cipher, s3msg-bus, s3confstore)

### PR DESCRIPTION
**Pull request by vimalraghwani20**
- merged s3cipher, s3confstore, s3msgbus installation script in to single setup.py file. 

-----
[View rendered s3cortxutils/README.md](https://github.com/Seagate/cortx-s3server/blob/br/vr/EOS-16667/s3cortxutils/README.md)
[View rendered s3cortxutils/s3cipher/s3cipher/README.md](https://github.com/Seagate/cortx-s3server/blob/br/vr/EOS-16667/s3cortxutils/s3cipher/s3cipher/README.md)
[View rendered s3cortxutils/s3msgbus/s3msgbus/README.md](https://github.com/Seagate/cortx-s3server/blob/br/vr/EOS-16667/s3cortxutils/s3msgbus/s3msgbus/README.md)

### CODE REVIEW

File: rpms/s3/s3rpm.spec [hunk 1]
```diff
@@ -380,16 +380,14 @@ rm -rf %{buildroot}
 %{py36_sitelib}/s3setup-%{version}-py?.?.egg-info
 %exclude %{py36_sitelib}/s3backgrounddelete/__pycache__/*
 %exclude %{py36_sitelib}/s3backgrounddelete/*.py
-%exclude %{py36_sitelib}/s3cortxutils/*.py
```
Comment (amitwac2608) [inline rpms/s3/s3rpm.spec]: why this needs to be removed ?

**Review by amitwac2608 — COMMENTED** [review]
seems good.

**Review by pratyush-seagate — APPROVED** [review]
+1

**Comment by gowthamchinna** [comment]
run mini-provisioning

**Pull request closed without merging**